src/status_im/contact/block.cljs

```diff
@@ -60,7 +60,8 @@
                                    (:contacts/contacts db)
                                    public-key)
                                   (assoc :blocked? true
-                                         :added?   false))
+                                         :added?   false
+                                         :active?  false))
```

Comment:
Should we set the `mutual?` attribute to `false` too ?

Reply:
@ibrkhalil  does not seem to have any effect but it can be set to false since the two contacts in this context are no longer mutual the moment one becomes blocked